.github/workflows/on-push-to-release-branch.yml

```diff
@@ -49,11 +75,24 @@ jobs:
         with:
           python-version: "3.9"
 
-      - name: Build package and run tests
+      - name: Build package
+        run: |
+          pip install -r requirements.txt
+          python -m build
+
+      - name: Run linters
+        id: linters
+        uses: ./.github/linters
```

Comment:
same nitpick as on previous pr, it might be more obvious what that file was for if you set up the directory structure as `./.github/actions/linters`.  Not a blocker if you disagree.

Reply:
Done, thanks for the suggestion.